### ISSUE TITLE
unable to create val.json

### ISSUE THREAD

**Issue by suravijayjilla** [kenshohara/3D-ResNets-PyTorch#249]
Hi,

Executed the code with the default dataset. Trained the model with kinetics dataset after successful training.
Unable to find a val.json for evaluation.
Please let me know how to generate the val.json file.

**Comment by guilhermesurek** [comment]
@suravijayjilla have you run the inference to generate the val.json? You should run it before calculate the top 1 accuracy.
![image](https://user-images.githubusercontent.com/23281984/115939168-9d2b7f80-a473-11eb-9e3b-c22357695650.png)

**Comment by YTHmamba** [comment]
> 

@ @guilhermesurek Hi, can only KINETICS data set calculate top1 accuracy? How is the UCF101 dataset calculated? I get an error using these two lines, and the ground_truth of the load_ground_truth function is 0 in util_scripts/ eval_specification. py
  
```
for video_id, v in data['database'].items():
        if subset != v['subset']:
            continue
        this_label = v['annotations']['label']
        ground_truth.append((video_id, class_labels_map[this_label]))
```
I think there is a mistake here. Do you know how to change this part of the code

**Comment by guilhermesurek** [comment]
@YTHmamba I didn't run in UCF101, but the code seems to be generic, independent of the dataset. Try to check if your inference json (or val.json) is similar to your ground truth json.  Or could be any difference in the class_labels_map.

**Comment by YTHmamba** [comment]
> @YTHmamba我没有在 UCF101 中运行，但代码似乎是通用的，独立于数据集。尝试检查您的推理 json（或 val.json）是否与您的基本事实 json 相似。或者可能是 class_labels_map 中的任何差异。

@guilhermesurek Thank you very much. Is the clip accuracy of the paper the identification accuracy of 16 frames? How is the accuracy of TOP1 video level calculated? What is the logic?
